phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/common/control/PhoenicisView.java

```diff
@@ -0,0 +1,49 @@
+package org.phoenicis.javafx.components.common.control;
+
+import javafx.beans.property.BooleanProperty;
+import javafx.beans.property.SimpleBooleanProperty;
+import org.phoenicis.javafx.components.common.skin.PhoenicisViewSkin;
+
+/**
+ * A base component for a view of the Phoenicis application
```

Comment:
I don't really understand the purpose of this class based on the description.

Reply:
Ok, I will base the description for the class on the description for `MainWindowView`.

Reply:
Done